apps/nft-service/src/nft/contract.ts

```diff
@@ -5,3 +5,13 @@ export async function getContract() {
   const signer = await getSigner("BrokerTransferAssistant");
   return connectNFT(signer);
 }
+
+export async function getMinterAddress() {
```

Comment:
This function is to allow both production to have a static, pre-defined address, but for dev & devlive to be able to use whatever addresses are current in the emulators